change/@fluentui-react-components-f34f712d-15c2-4cc9-be35-95c55efad70b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat(react-components): remove mergeThemes from public API",
```

```suggestion
  "comment": "remove mergeThemes from public API",
```

_nit_ As changelog is grouped there is no sense to include package name to a changelog entry

---

I see, hmm...

for better context, this is my muscle memory from OSS that used conventional changelog most of the time. Also when selecting commits via beachball CLI it makes it very explicit what commit is affecting that change otherwise one needs to use significant cognitive load to pick proper thing, I'll discuss this with folks if/how can we make this better.